blackbox/docs/admin/privileges.rst

```diff
@@ -109,15 +121,33 @@ and functions of the ``doc`` schema::
     on a schema level, CrateDB takes the schema name provided without further
     validation.
 
-This statement will deny ``DQL`` privilege to user riley on the ``doc`` schema
-table ``doc.accounting``. However, user riley, will still have ``DQL``
-privilege on all the other tables of the ``doc`` schema.
-
 .. NOTE::
 
     Privileges can be managed on all schemas and tables of the cluster,
     except the ``information_schema``.
 
+Views are on the same hierarchy with tables, i.e. a privilege on a view
+is gained through a ``GRANT`` on either the view itself, the schema the view
+belongs to, or a cluster-wide privilege. Privileges on tables which are
+referenced in the view do not grant any privileges on the view itself. On the
+contrary, even if the user does not have any privileges on a view's referenced
+tables but on the view itself, the user can still access the tables through
+the view. For example::
+
+    cr> CREATE VIEW first_customer as SELECT * from doc.accounting ORDER BY id LIMIT 1
+    CREATE OK, 1 row affected (... sec)
+
+Previously we had issued a ``DENY`` for user ``riley`` on ``doc.accounting`` but we
+can still access it through the view::
+
+    cr> Select id from first_customer;
```

Comment:
Maybe uppercase keywords like `SELECT` and `FROM` everywhere to make it consistent. Right now some are uppercased and some not.